docs/release-notes.md

```diff
@@ -33,15 +33,26 @@
 
 # Release notes
 
+## Trino Gateway 7 (19 March 2024)
+
+[JAR file gateway-ha-7-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/7/gateway-ha-7-jar-with-dependencies.jar)
+[Docker container trinodb/trino-gateway:7](https://hub.docker.com/layers/trinodb/trino-gateway/7)
+
+* Replace user interface with a new modern UI. ([#116](https://github.com/trinodb/trino-gateway/pull/116))
+* Add a light-weight health check of the Trino cluster using the `v1/info` endpoint. ([#264](https://github.com/trinodb/trino-gateway/pull/264))
+* Fix query tracking based on request querystring parsing. ([#265](https://github.com/trinodb/trino-gateway/pull/265))
+
+[Details about all pull requests and issues](https://github.com/trinodb/trino-gateway/issues?q=milestone%3A7+is%3Aclosed)
+
 ## Trino Gateway 6 (16 Feb 2024)
 
-[gateway-ha-6-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/6/gateway-ha-6-jar-with-dependencies.jar)
+[JAR file gateway-ha-6-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/6/gateway-ha-6-jar-with-dependencies.jar)
+[Docker container trinodb/trino-gateway:6](https://hub.docker.com/layers/trinodb/trino-gateway/6)
```

Comment:
This link doesn't work. (404)

Reply:
Thanks for catching .. I removed usage of a link for these since dockerhub requires the full sha ... which I cant know before the release.. and its also kinda not needed.

Reply:
removed links